src/discussions/comments/CommentsView.jsx

```diff
@@ -67,8 +67,8 @@ function DiscussionCommentsView({
     isLoading,
     handleLoadMoreResponses,
   } = usePostComments(postId, endorsed);
-  const sortedComments = useMemo(() => orderBy(comments, ['endorsed', 'createdAt'],
-    ['desc', 'desc']));
+  const sortedComments = useMemo(() => orderBy(comments, ['endorsed'],
```

Comment:
Is sort based on endorsed working? What is the difference in ui behavior if we remove that as well?


Is it to get endorsed responses on top on initial load?
I think we are also working on shifting that logic to backend in another ticket.

Reply:
Yes... Endorsed responses will be on top...
Frontend will be changed when the change will be implemented on backend

Reply:
We do not need to sortBy desc order. we can split the comments array into 2 arrays. one for endorse and one for un endorse and render these arrays one by one which will persist the API response order.  A filter function is available in utils file.